doc/lsst.pipe.tasks/tasks/lsst.pipe.tasks.calibrateImage.CalibrateImageTask.rst

```diff
@@ -4,10 +4,10 @@
 CalibrateImageTask
 ##################
 
-`~lsst.pipe.tasks.calibrateImage.CalibrateImageTask` performs "single frame processing" on one (single *visit*) or two (two *snap* visit) post- :ref:`Instrument Signature Removal <lsst.ip.isr>` single detector exposure (``postISRCCD``).
+`~lsst.pipe.tasks.calibrateImage.CalibrateImageTask` performs "single frame processing" on one (single *visit*) or two (two *snap* visit) post- :ref:`Instrument Signature Removal <lsst.ip.isr>` single detector exposure (``post_isr_image``).
 This involves merging two *snaps* (if provided) into one *visit* exposure, repairing cosmic rays and defects, detecting and measuring sources on the exposure to make an initial estimation of the point spread function (PSF), using that same catalog of psf stars to compute the astrometric calibration, then re-doing detection and measurement with the fitted PSF to compute the photometric calibrations, and computing summary statistics of the exposure and measured catalog.
-Its primary outputs are a calibrated, background-subtracted exposure (``initial_pvi``, pixel values in nanojansky) and catalog (``initial_stars_detector``) of bright, well-measured point-like sources that were used as inputs to calibration and that are suitable for downstream use (for example as kernel candidates in difference imaging).
-This task replaces the two older tasks `~lsst.pipe.tasks.characterizeImage.CharacterizeImageTask` (roughly repair/estimate PSF/aperture correct) and `~lsst.pipe.tasks.calibrate.CalibrateTask` (roughly detect/measure/astrometry/photometry).
+Its primary outputs are a calibrated, background-subtracted exposure (``preliminary_visit_image``, pixel values in nanojansky) and catalog (``single_visit_star_unstandardized``) of bright, well-measured point-like sources that were used as inputs to calibration and that are suitable for downstream use (for example as kernel candidates in difference imaging).
+This task replaces the two older tasks `~lsst.pipe.tasks.characterizeImage.CharacterizeImageTask` (roughly: repair/estimate PSF/aperture correct) and `~lsst.pipe.tasks.calibrate.CalibrateTask` (roughly: detect/measure/astrometry/photometry).
```

Comment:
I would just drop this sentence, since these older tasks have been removed.

Reply:
They are not used in pipelines any more, but the code is still there and they have not yet been deprecated: [DM-49564](https://rubinobs.atlassian.net/browse/DM-49564) is still ToDo. We still have people asking about them occasionally on slack, so I think we need this reference at least until they've been deprecated, but preferably until they've been removed.